interface.go

```diff
@@ -0,0 +1,36 @@
+package indexer
+
+import (
+	"github.com/filecoin-project/go-indexer-core/entry"
+	"github.com/ipfs/go-cid"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+type Interface interface {
+	// Get retrieves a slice of entry.Value for a CID
+	Get(c cid.Cid) ([]entry.Value, bool, error)
```

Comment:
consider making `entry.Value` also in this package so that components working against the interface only need to import one package?

Reply:
Probably a good idea.

Reply:
Done.  It also just makes sense to have that in the same package as the interface since it is part of the interface definition.